app/dev/styles/main.scss

```diff
@@ -0,0 +1,64 @@
+html{
+  height:100%;
+}
+
+body{
+  height: 97.5%;
+  margin: 0
+}
+
+.main {
+  background-color: #E5EAEC;
+  background-image: url('http://localhost:9292/img/bg-tl.png'), url('http://localhost:9292/img/bg-br.png');
```

Comment:
Can we do that without hard-coding a port? If we run on a different port this will break.